### PR TITLE
1052 - Implement new task editor 4

### LINKED ISSUE #1052: Implement new task editor
image

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1052

https://github.com/google/ground-platform/assets/10047945/e64bb50a-ef44-46a6-87e5-73b125faafb0

### CODE REVIEW

File: web/src/app/pages/create-survey/task-details/task-input/edit-option/edit-option.component.html [hunk 1]
```diff
@@ -32,15 +32,15 @@
         </ng-container>
       </mat-error>
     </mat-form-field>
-    <mat-form-field class="code-option">
+    <!-- <mat-form-field class="code-option">
```
Comment (os-micmec) [inline web/src/app/pages/create-survey/task-details/task-input/edit-option/edit-option.component.html:35]: Can we remove this?

File: web/src/app/pages/main-page-container/main-page/job-dialog/task-editor/_task-editor.component-theme.scss [hunk 2]
```diff
@@ -27,13 +27,13 @@
 
   // TODO(amysorto): use typography-config once typography levels defined
 
-  .task-type {
-    font-size: 12px;
-  }
+  // .task-type {
```
Comment (os-micmec) [inline web/src/app/pages/main-page-container/main-page/job-dialog/task-editor/_task-editor.component-theme.scss:30]: Can we remove this too?

**Review by os-micmec — APPROVED** [review]

File: web/src/app/pages/main-page-container/main-page/job-dialog/task-editor/_task-editor.component-theme.scss (hunk 2 above)
Comment (gino-m) [reply web/src/app/pages/main-page-container/main-page/job-dialog/task-editor/_task-editor.component-theme.scss:30]: > Can we remove this too?

Looks like the PR is still in Draft state (work in progress)

Comment (os-micmec) [reply web/src/app/pages/main-page-container/main-page/job-dialog/task-editor/_task-editor.component-theme.scss:30]: Ah my bad!

**Review by gino-m — APPROVED** [review]
LGTM! @os-micmec or @DaoyuT PTAL?

**Review by os-micmec — APPROVED** [review]

**Pull request merged**